pandas/core/reshape/pivot.py

```diff
@@ -429,8 +429,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
 
     >>> foo = pd.Categorical(['a', 'b'], categories=['a', 'b', 'c'])
     >>> bar = pd.Categorical(['d', 'e'], categories=['d', 'e', 'f'])
-    >>> crosstab(foo, bar)  # 'c' and 'f' are not represented in the data,
-    ...                     # but they still will be counted in the output
+    >>> crosstab(foo, bar, dropna=False)  # 'c' and 'f' are not represented
```

Comment:
@testvinder , just a suggestion, you could've actually just changed the output regarding the given code and mentioned in comments about the default value of `dropna`being set to `True`

Reply:
I would actually just add an example instead of modifying this one.

Reply:
@uds5501 I actually think it's better to set ´dropna` to `False`. The whole point of specifying the extra categories is to include them in the crosstab. It's a small change but the original is very confusing because the output cannot be reproduced.

Reply:
@testvinder : Ah, yes, your point is well-taken.  Can you update the original example with the correct output and add your example with `dropna=False` ?

Reply:
@gfyoung Sure, will do.